### PR TITLE
fix: do not reduce count for subadmins if they are members of group

### PR DESCRIPTION
**Pull request by Antreesy**
* Follow-up to #48437

## Summary

Counter on sidebar doesn't reflect the actual number of group members, if subadmin is a member too.
For each group, counter is reduced by 1, and for 'All accounts', counter is reduced by amount of groups subadmin is in.

There is a comment:
> we prevent subadmins from looking up themselves so we lower the count of the groups he belongs to

but I don't see a point of it, since we fetch and show yourself in the list anyway, so just removing inconsistency

Before | After
-- | --
image | image
_Subadmin: 'alice', member of 'test group' and 'test group 2'_

## TODO

- [ ] ...

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/lib/Controller/UsersController.php [hunk 1]
```diff
@@ -156,10 +156,10 @@ public function usersList(): TemplateResponse {
...
 					// we prevent subadmins from looking up themselves
 					// so we lower the count of the groups he belongs to
-					if (array_key_exists($group['id'], $userGroups)) {
-						$groups[$key]['usercount']--;
-						$userCount -= 1; // we also lower from one the total count
-					}
+					// if (array_key_exists($group['id'], $userGroups)) {
+					// $groups[$key]['usercount']--;
+					// $userCount -= 1; // we also lower from one the total count
+					// }
```
Comment (provokateurin) [inline apps/settings/lib/Controller/UsersController.php:162]: Just get rid off the whole thing, I agree it doesn't make much sense to do these subtractions.

**Review by provokateurin — CHANGES_REQUESTED** [review]

**Review by provokateurin — APPROVED** [review]

**Review by blizzz — APPROVED** [review]

**Pull request merged**